flexget/components/thetvdb/api_tvdb.py

```diff
@@ -212,11 +212,12 @@ def __init__(self, tvdb_id, language):
         self._genres = [TVDBGenre(id=name) for name in series['genre']] if series['genre'] else []
 
         if self.first_aired is None:
+            log.debug('Falling back to getting first episode aired date for series %s', self.name)
             try:
-                episode = TVDBRequest().get('series/%s/episodes/query?absoluteNumber=1' % self.id, language=language)
+                episode = TVDBRequest().get('series/%s/episodes/query?airedSeason=1&airedEpisode=1' % self.id, language=language)
```

Comment:
What if the first aired episode is a special?

Reply:
So I just got back and was about to start implementing change to get first episode that aired, however I found an example that shows that TheTVDB uses the first season episode as its official air date.
https://www.thetvdb.com/series/good-witch

So this suggests that we should only use 1x01 air date.